static/js/controllers/reports.js

```diff
@@ -297,23 +297,11 @@ var _ = require('underscore'),
            ) {
           $scope.filtered = true;
           liveList = LiveList['report-search'];
-          liveList.set([]);
```

Comment:
Was this line unnecessary?  Or will previous searches now get mixed up with new search content?